media-gfx/openvdb/metadata.xml

```diff
@@ -19,9 +19,9 @@
 		Chance of Meatballs2' and 'How to Train Your Dragon 2'. 
 	</longdescription>
 	<use>
-		<flag restrict="&gt;=media-gfx/openvdb-4.0.0" name="abi3-compat">
-			Enables compatibility with the old 3.x ABI.
-			enabled by dfault.
+		<flag name="abi3-compat">
+			Disables newer features to maintain compatibility with ABI3.
+			Enabled by dfault.
```

Comment:
Still haven't fixed that that typo. ;) And is there any reason for the line break?

Reply:
Fixed. :)